tests/test_token_blacklist.py

```diff
@@ -237,25 +237,25 @@ def setUp(self):
 
         super().setUp()
 
+    @override_api_settings(BLACKLIST_AFTER_ROTATION=True)
     def test_token_verify_serializer_should_honour_blacklist_if_blacklisting_enabled(
         self,
     ):
-        with override_api_settings(BLACKLIST_AFTER_ROTATION=True):
-            refresh_token = RefreshToken.for_user(self.user)
-            refresh_token.blacklist()
+        refresh_token = RefreshToken.for_user(self.user)
+        refresh_token.blacklist()
 
-            serializer = TokenVerifySerializer(data={"token": str(refresh_token)})
-            self.assertFalse(serializer.is_valid())
+        serializer = TokenVerifySerializer(data={"token": str(refresh_token)})
+        self.assertFalse(serializer.is_valid())
 
+    @override_api_settings(BLACKLIST_AFTER_ROTATION=False)
```

Comment:
Is this really needed since `BLACKLIST_AFTER_ROTATION` is default to `False`, maybe for explicitly sake.